static-site/src/components/People/teamMembers.js

```diff
@@ -0,0 +1,127 @@
+export const teamMembers = {
```

Comment:
P.S. This is the file to modify -- please add a "blurb" key to your entry!

Reply:
Done for all teamMembers now